string slot
string value
float64 probability